interface.yaml

```diff
@@ -0,0 +1,12 @@
+name: bind-client
+summary: Interface for intergrating bind
+maintainer: DevOps Centres
+ignore:
+  - 'unit_tests'
```

Comment:
nitpick: are quotes needed?

Reply:
I'm not sure if the quotes are needed, but all the examples that I found are using it.

Reply:
I've tried to remove the quotes and it seems that works normally. Will remove it.